recipes/pickle5/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pickle5" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2dd3376b4b8b6ab8c51a19d4915b781c98aa117065e4d32713187ae5bee90b50
+
+build:
+  number: 0
+  skip: true  # [py<36]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pickle5
+  commands:
+    - python -m pickle5.test.test_pickle
+
+about:
+  home: https://github.com/pitrou/pickle5-backport
+  license: PSF 2
+  #license_file: LICENSE
```

Comment:
License is currently not packaged on PyPI. Have raised issue ( https://github.com/pitrou/pickle5-backport/issues/2 ) on this.

Reply:
Addressed by updating to 0.0.4.